### PR TITLE
DOCS-5488 overhaul /customization/organization-profile page

### PR DESCRIPTION
**Pull request by alexisintech**
Recently, the /customization/user-profile docs page got a complete overhaul. Details are in this PR: https://github.com/clerk/clerk-docs/pull/751

We need to also update the /customization/organization-profile docs page for consistency!

### CODE REVIEW

**Comment by S3Prototype** [comment]
Unrelated to this PR, but this preview made me realize how tight the line height on our paragraphs is. Look how bunched up these lines look with the words wrapped in code tags:
image

File: docs/components/customization/organization-profile.mdx [hunk 1]
```diff
@@ -1,416 +1,360 @@
...
-      ```
-    </CodeBlockTabs>
-  </Tab>
-
-  <Tab>
-    ```jsx filename="/organization-profile.tsx"
-    import { OrganizationProfile } from "@clerk/clerk-react";
-    import { CustomProfilePage, CustomTerms } from "../components";
-    import { CustomIcon } from "../icons";
+## Add a custom page to the `<OrganizationProfile />` component
```
Comment (S3Prototype) [inline docs/components/customization/organization-profile.mdx:23]:
```suggestion
## Add a custom page to `<OrganizationProfile />`
```

While these headings are probably nice for search performance, they look super crowded in the ToC:

image

This and the other change suggestions just make the heading a little less crowded

File: docs/components/customization/organization-profile.mdx [hunk 2]
```diff
@@ -1,416 +1,360 @@
...
+      </OrganizationProfile.Page>
+    </OrganizationProfile>
+  );
+
+  export default OrganizationProfilePage;
+  ```
+</CodeBlockTabs>
+
+
+## Add a custom link to the `<OrganizationProfile />` component
```
Comment (S3Prototype) [inline docs/components/customization/organization-profile.mdx:157]:
```suggestion
## Add a custom link to `<OrganizationProfile />`
```

See [my earlier comment ](https://github.com/clerk/clerk-docs/pull/768/files#r1516555049)

File: docs/components/customization/organization-profile.mdx [hunk 3]
```diff
@@ -1,416 +1,360 @@
...
+    </OrganizationProfile>
+  );
+
+  export default OrganizationProfilePage;
+  ```
+</CodeBlockTabs>
 
 ## Advanced use cases
 
 ### Reordering default routes
```
Comment (S3Prototype) [inline docs/components/customization/organization-profile.mdx:241]:
```suggestion
## Reordering default routes
```

**Comment by alexisintech** [comment]
> Unrelated to this PR, but this preview made me realize how tight the line height on our paragraphs is. Look how bunched up these lines look with the words wrapped in code tags: image

this is getting resolved in the docs redesign project.

**Comment by alexisintech** [comment]
these suggestions will need to be applied to the user profile overhaul, as they weren't brought up in that preceding pr - i'll create another one

**Pull request merged**